Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -241,89 +228,18 @@ private void updateSmartDashboard( )
   public void driveWithLimelightInit(Vision vision, boolean m_endAtTarget)
   {
     // get pid values from dashboard
-    m_turnConstant = SmartDashboard.getNumber("DLL_turnConstant", m_turnConstant);
-    m_turnPidKp = SmartDashboard.getNumber("DLL_turnPidKp", m_turnPidKp);
-    m_turnPidKi = SmartDashboard.getNumber("DLL_turnPidKi", m_turnPidKi);
-    m_turnPidKd = SmartDashboard.getNumber("DLL_turnPidKd", m_turnPidKd);
-    m_turnMax = SmartDashboard.getNumber("DLL_turnMax", m_turnMax);
-
-    m_throttlePidKp = SmartDashboard.getNumber("DLL_throttlePidKp", m_throttlePidKp);
-    m_throttlePidKi = SmartDashboard.getNumber("DLL_throttlePidKi", m_throttlePidKi);
-    m_throttlePidKd = SmartDashboard.getNumber("DLL_throttlePidKd", m_throttlePidKd);
-    m_throttleMax = SmartDashboard.getNumber("DLL_throttleMax", m_throttleMax);
-    m_throttleShape = SmartDashboard.getNumber("DLL_throttleShape", m_throttleShape);
-
-    m_targetAngle = SmartDashboard.getNumber("DLL_targetAngle", m_targetAngle);
-    m_setPointDistance = SmartDashboard.getNumber("DLL_setPointDistance", m_setPointDistance);
-    m_angleThreshold = SmartDashboard.getNumber("DLL_angleThreshold", m_angleThreshold);
-    m_distThreshold = SmartDashboard.getNumber("DLL_distThreshold", m_distThreshold);
-
-    // load in Pid constants to controller
-    m_turnPid = new PIDController(m_turnPidKp, m_turnPidKi, m_turnPidKd);
-    m_throttlePid = new PIDController(m_throttlePidKp, m_throttlePidKi, m_throttlePidKd);
 
     vision.m_tyfilter.reset( );
     vision.m_tvfilter.reset( );
     vision.syncStateFromDashboard( );
   }
 
-  public void driveWithLimelightExecute(Vision vision)
```

Comment:
If you're leaving in some commented code for examples, probably want to at least leave in an empty driveWithLimelightExecute{} method which will be called during the command.